### ISSUE TITLE
[FEATURE REQUEST] Choose action that is performed when Automatic Cleaning runs or "Clean" button is pressed

### ISSUE THREAD

**Issue by Agych** [Cookie-AutoDelete/Cookie-AutoDelete#757]
**Is your feature request related to a problem? Please describe.**

Option for setting the action that is performed when Automatic Cleaning runs or "Clean" button is pressed.

**Describe the solution you'd like**

It would be great, if in the Settings it was possible to choose between the following options:
- Clean, exclude domains from open tabs.
- Clean, include domains from open tabs.
(this would ofc ignore the domains listed in the white/gray lists)

**Comment by kennethtran93** [comment]
There is already an option (dropdown next to clean) to clean cookies including open tabs.

Implementing this setting would require additional coding to swap the button/dropdown buttons.

In addition, I'm implementing a right-click feature that will have cleaning options available, so that should be better...maybe.

**Comment by Agych** [comment]
> There is already an option (dropdown next to clean) to clean cookies including open tabs.

Yes I know, the goal is to have the favourite/most-used operation chosen by the user in the settings and executed in one-click by pressing "Clean", without the need to do to the dropdown. :-) 
So for example I would go to the settings, choose my fav as "Clean, include domains from open tabs", and the get it executed in a single click, and without the need to do to the dropdown.

> What were you aiming for with this portion of the request...I'm not sure why you want automatic cleaning to change the action otherwise that will confuse the predefined actions for each list?

Aiming for simply being able to have all site data (including open tabs, but excluding domains in white/gray lists) cleaned. In other words - all unwanted site data get cleaned.
This could be performed for example on a tab-close
(or it could be even performed periodically, with the period defined in the settings with the options like 1/3/5/10/60/etc minutes).

**Comment by kennethtran93** [comment]
I'm implementing a right-click with cleaning options that can be triggered on the page / icon.  Maybe that may be of use for you in the near future?
![Screenshot_20200601_093628](https://user-images.githubusercontent.com/6724477/83693612-442cd180-a5ab-11ea-84f2-7545a86668d4.png)
